c2corg_ui/static/js/utils.js

```diff
@@ -123,6 +123,26 @@ app.utils.formatDate = function(date) {
 
 
 /**
+ * check that the start date if before end date.
+ * Return true is start > end
+ * @export
+ * @param {Object} outing
+ * @return {(boolean | undefined)}
```

Comment:
I mean that in this case i'm never interested in `false`, only `true` interests me. That's why I wanted true or nothing. I have already corrected it to return false as well, but it's of no use in this particular case.